### PR TITLE
Documentation for province report

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for issue #1570

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 36 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 31 sec

File: docs/src/user/manual/explore/reports.md
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/reports.md:1]: new changes look good, but I missed the use of "your" and "you" throughout this doc. Please change those to be "users", "the user", "their", etc.

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 44 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 30 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 1 min 24 sec

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 6 min 48 sec

**Pull request merged**